extension/src/plots/errors/collect.ts

```diff
@@ -30,13 +30,17 @@ export const collectErrors = (
   ]
 }
 
-const getMessage = (error: PlotError): string => {
+export const getMessage = (error: PlotError): string => {
```

Comment:
[F] Decided that I had to change the format of this now. When I push the transformation logic out to the clients (tree, etc) I'll update to transform the errors as they come in.

Reply:
It will simplify all of the errors logic but requires updating all of the tests so will break the change out.